_posts/2016-10-14-sarah-vang.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Not Enough"
+category: ourstories
+tags: 
+author: Sarah Vang
+featured-image: '/static/images/featured/2016-10-14-sarah-vang.jpg'
+featured-alt-text: "Side portrait of an Asian woman with photoshopped black smoke behind her. MYAAPIVOTE is for educational justice is at the bottom of the image."
+tweet: "Read YTEAMer Sarah Vang of @18millionrising share what her vote is for #MyAAPIVote"
+facebook: "Read YTEAMer Sarah Vang of @18millionrising share what her vote is for #MyAAPIVote"
+blurb: "My parents made sure to work hard, so they can provide food, shelter and opportunity for my siblings and me. They believe that as long as you work hard in America, you could achieve the American dream. But they’re wrong; hard work is not enough."
+---
+
+I am a daughter of Hmong refugees who escaped from genocide faced by our people after the Secret War in Laos. My parents fled to the United States when my oldest sister was still a baby. With no educational diploma or degree, my parents worked entry-level jobs at manufacturing companies and Thai/Chinese restaurants. Nevertheless, my parents made sure to work hard, so they can provide food, shelter and opportunity for my siblings and me. <strong>They believe that as long as you work hard in America, you could achieve the American dream. But they’re wrong; hard work is not enough.</strong>
+
+It’s not enough that my dad works 12-hour shifts in a kitchen for ten years, because he still has not receive a raise or promotion despite his dedication to deliver his best performance and punctuality every single day. It’s not enough that my mom works crappy shifts at a manufacturing company where she stands for long hours, only to come home to complain about how much her knees and back hurt. She refuses to go to the hospital because medical bills are too high, so she has either one of my sisters or me to massage her with a minty ointment in hopes that her pain goes away so she can rest and go back to work in early morning the next day. It’s not enough that my parents never miss a payment for bills and taxes, because my parents were not considered U.S. citizens until they participated in the rigged naturalization process to gain their right to vote. And even now “earning” the right to vote is not enough, because my parents have trouble understanding why their voice matters in a country, in which the Hmong presence is invisible to even when the CIA was the main reason why we’re here in the first place.
```

Comment:
Simple copy typos in this para: "because he still has not receive_d_ a raise"; "Hmong presence is invisible to..."